server.js

```diff
@@ -89,102 +90,107 @@ function loginSession(req, res, key) {
     });
 }
 
+// -------------- SHOW THE ADMIN PAGE ----------------------------
+
+// we are making an assumption here, that you have a good API key with read/write access
 function adminPage(req, res) {
-  let URL = 'https://api.jotform.com/user/forms';
-  let key = req.cookies.jotform;
+  let apiKey = req.cookies.jotform;
+  let URL = `https://api.jotform.com/user?apiKey=${apiKey}`;
 
   superagent.get(URL)
-    .set('APIKEY', key)
     .then(result => {
-      let forms = result.body.content.filter(form => {
-        if(form.status === 'ENABLED') {
-          let theForm = new Form(form);
-          console.log(theForm);
-          return theForm;
-        }
-      })
-      res.render('pages/admin', {forms:forms});
+      let limit = result.body['limit-left'];
+      console.log('limit: ', limit);
+
+      let URL = 'https://api.jotform.com/user/forms';
+      superagent.get(URL)
+        .set('APIKEY', apiKey)
+        .then(result => {
+          let forms = result.body.content.filter(form => {
+            if(form.status === 'ENABLED') {
+              let theForm = new Form(form);
+              return theForm;
+            }
+          })
+          res.render('pages/admin', {forms:forms, limit:limit});
+        })
+        .catch(err => console.error(err));
     })
     .catch(err => console.error(err));
 }
 
+// ------------ SHOW THE GRAPH OF A SURVEY ----------------------
+
 function showResult(req, res) {
   let id = req.params.id;
   let key = req.cookies.jotform;
-
   let URL = `https://api.jotform.com/form/${id}/submissions?apiKey=${key}`;
 
-  let title = req.body.newSurvey;
-
-//   let cloneURL = `https://api.jotform.com/form/${TEMPLATE_FORM}/clone?apiKey=${key}`;
-
-//   superagent.post(cloneURL)
-//     .then( result => {
-
-//       let id = result.body.content.id;
-
-//       let setTitleURL = `https://api.jotform.com/form/${id}/properties?apiKey=${key}`;
-
-//       superagent.put(setTitleURL)
-//         .send( { 'properties' : { 'pagetitle' : title }} )
-//         .then( () => {
-//           res.redirect('/admin');
-//         })
-//         .catch(err => {
-//           console.error(err);
-//         });
-
-//     })
-//     .catch(err => {
-//       console.error(err)
-//     });
-
-// }
-
-
+  // Collect all the submissions for a given form
   superagent.get(URL)
     .then(result => {
       let submissions = result.body.content;
 
 
-      // this is the total question asked, we'll set that value in reduce below
+      // this is the total numbers of questions asked, we'll set that value in reduce below, so we can use it even later
       let total = 0;
       // create an array of each persons sum total of TRUE answers
       let people = submissions.map( person => {
         let keys = Object.keys(person.answers);
         return keys.reduce((acc, key, idx)=>{
```

Comment:
Make more readable. This is efficient but it is hard to understand what is going on in the code block